src/Components/ADT3DSceneBuilder/Internal/VisualRuleForm/Internal/ConditionsCallout/ConditionsCallout.tsx

```diff
@@ -0,0 +1,255 @@
+            });
+        },
+        [conditionCalloutDispatch]
+    );
+
+    const onValuesChange = useCallback(
+        (
+            valueType: IDTDLPropertyType,
+            newValues: unknown[],
+            index?: number
+        ) => {
+            if (valueType === 'boolean' || valueType === 'string') {
+                if (newValues.length) {
+                    validityMap.current.ranges = true;
+                } else {
+                    validityMap.current.ranges = false;
+                }
+                conditionCalloutDispatch({
+                    type: ConditionCalloutActionType.FORM_CONDITION_VALUES_SET,
+                    payload: { values: newValues }
+                });
+            } else {
+                const rangeValues = conditionCalloutState.conditionToEdit.values
+                    ? deepCopy(conditionCalloutState.conditionToEdit.values)
+                    : [0, 0];
+                // Values
+                rangeValues[index] = newValues[0];
+                validityMap.current.ranges = areRangesValid(
+                    rangeValues,
+                    valueRangeType
+                );
+                conditionCalloutDispatch({
+                    type: ConditionCalloutActionType.FORM_CONDITION_VALUES_SET,
+                    payload: { values: rangeValues }
+                });
+            }
+        },
+        [conditionCalloutState.conditionToEdit.values, valueRangeType]
+    );
+
+    const onActionValueChange = useCallback(
+        (key: ActionItemKey, value: string) => {
+            if (key === 'color') {
+                conditionCalloutDispatch({
+                    type: ConditionCalloutActionType.FORM_CONDITION_COLOR_SET,
+                    payload: { color: value }
+                });
+            } else {
+                conditionCalloutDispatch({
+                    type: ConditionCalloutActionType.FORM_CONDITION_ICON_SET,
+                    payload: { iconName: value }
+                });
+            }
+        },
+        []
+    );
+
+    const handleSaveClick = useCallback(() => {
+        onSave(conditionCalloutState.conditionToEdit);
+        onDismiss();
+    }, [onDismiss, onSave, conditionCalloutState.conditionToEdit]);
+
+    const handleCancelClick = useCallback(() => {
+        onDismiss();
+    }, [onDismiss]);
+
+    // styles
+    const classNames = getClassNames(styles, {
+        theme: useTheme()
+    });
+
+    return (
+        <>
+            {isOpen && (
+                <Callout
+                    target={target}
+                    styles={classNames.subComponentStyles.callout}
+                    onDismiss={onDismiss}
+                    directionalHint={DirectionalHint.rightCenter}
+                >
+                    <Stack tokens={{ childrenGap: 12 }}>
+                        {calloutType === CalloutInfoType.create ? (
+                            <h4 className={classNames.title}>
```

Comment:
nit: instead of duplicating the whole h4 tag here, can we just switch the string we include?